src/CNLOADS.CPP

```diff
@@ -3381,7 +3382,7 @@ RC RSYS::rs_EndSubhr()
 	// verify RSYSRES_IVL_SUB layout at compile time
 	// fixed sequence allows array access by rs_mode (see code below)
 	// rsmHEAT/rsmCOOL/rsmOAV definitions must be consistent with member sequences.
-#define QZONECHK( m, oDif) static_assert( &((RSYSRES_IVL_SUB *)0)->m-&((RSYSRES_IVL_SUB *)0)->qhZoneSen == oDif, "Bad seq " #m)
+#define QZONECHK( m, oDif) assert( &((RSYSRES_IVL_SUB *)0)->m-&((RSYSRES_IVL_SUB *)0)->qhZoneSen == oDif && "Bad seq " #m)
```

Comment:
assert() is runtime.  static_assert() is compile time.  This was deliberately coded to be a compile time check so the runtime cost is not incurred every subhour.  Can a method be found that does the compile time check and is OK on all compilers?

Reply:
I can look for a replacement. I was worried if that would be a problem. The error that I got was that it was non-constant condition. `non-constant condition for static assertion`.

Reply:
You could try &((const RSYSRES_IVL_SUB *)0)->m - &((const RSYSRES_IVL_SUB *)0)->qhZoneSen

That is, add consts.